english/src/in-which-countries-is-it-available.md

```diff
@@ -52,21 +75,31 @@ Here we present you the list of currencies:
 | PYG  | Paraguayan Guarani          | 🇵🇾  |
 | QAR  | Qatari Rial                 | 🇶🇦  |
 | RON  | Romanian Leu                | 🇷🇴  |
+| RSD  | Serbian Dinar               | 🇷🇸  |
 | RUB  | Russian Ruble               | 🇷🇺  |
+| RWF  | Rwandan Franc               | 🇷🇼  |
 | SAR  | Saudi Riyal                 | 🇸🇦  |
 | SEK  | Swedish Krona               | 🇸🇪  |
 | SGD  | Singapore Dollar            | 🇸🇬  |
+| THB  | Thai Baht                   | 🇹🇭  |
+| TND  | Tunisian Dinar              | 🇹🇳  |
 | TRY  | Turkish Lira                | 🇹🇷  |
 | TTD  | Trinidad and Tobago Dollar  | 🇹🇹  |
+| TWD  | New Taiwan Dollar           | 🇹🇼  |
+| TZS  | Tanzanian Shilling          | 🇹🇿  |
 | UAH  | Ukrainian Hryvnia           | 🇺🇦  |
+| UGX  | Ugandan Shilling            | 🇺🇬  |
 | USD  | US Dollar                   | 🇺🇸  |
 | UYU  | Peso uruguayo               | 🇺🇾  |
+| UZS  | Uzbekistan Som              | 🇺🇬  |
 | VES  | Bolívar                     | 🇻🇪  |
 | VND  | Vietnamese Dong             | 🇻🇳  |
+| XAF  | CFA Franc BEAC              | 🇨🇲🇨🇫🇨🇬🇬🇦🇬🇶🇹🇩  |
```

Comment:
XAF and XOF are being used both in Chad? acording with wikipedia they are using XAF

Reply:
That is correct. Tchad is using XAF.